src/frontend/js/widgets/Dashboard/components/DashboardItem/Contract/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button } from '@openfun/cunningham-react';
+import { Icon, IconTypeEnum } from 'components/Icon';
+import { Contract } from 'types/Joanie';
+import { DashboardItem } from 'widgets/Dashboard/components/DashboardItem/index';
+import ContractStatus from './components/ContractStatus';
+
+interface DashboardItemContractProps {
+  contract: Contract;
+}
+export const DashboardItemContract = ({ contract }: DashboardItemContractProps) => {
+  // TODO: download action
+  const onDownload = () => {
+    alert('TODO: download action');
+  };
+
+  // TODO: sign action
+  const onSign = () => {
+    alert('TODO: sign action');
```

Comment:
I think we might wait a bit for the sign contract PR to be merged in order to go ahead with this one

Reply:
This PR is made to be merged before or after the sign implementation.
I don't think we should wait if it's ready.

Reply:
Allright, let's move on